build-docker.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+#
+# prerequisites:
+#  * create the bxd namespace
+#  * create the docker-login secret in the bxd namespace
+#  * rabbit container running in rabbit-mq-ha.rabbit
+#
+#
+
+
+current_mvn_version=$(mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec);
+export MVN_VERSION=${1:-$current_mvn_version}
```

Comment:
`MVN_VERSION` seems a little bit misleading, as it is actually the CoatRack version. 

So maybe use `COATRACK_VERSION`? Or maybe use a more generic name like `VERSION_TO_BUILD`, as `MVN_VERSION` is used "all over the place" right now, so maybe a more generic name would be better to facilitate re-use!?

Reply:
its actually referring to the maven version

Reply:
Just to be clear, when running this locally, `MVN_VERSION` is set to `project.version` from `pom.xml`. So it is not the version of the locally installed mvn, but the CoatRack version from mvn pom, right?